colossus/src/main/scala/colossus/core/Worker.scala

```diff
@@ -157,11 +156,10 @@ private[colossus] class Worker(config: WorkerConfig) extends Actor with ActorLog
 
   private val workerIdTag = Map("worker" -> (io.name + "-" + workerId.toString))
 
-  import config.io.metrics.base
-
-  val eventLoops              = Rate(io.namespace / "worker" / "event_loops")
-  val numConnections          = Counter(io.namespace / "worker" / "connections")
-  val rejectedConnections     = Rate(io.namespace / "worker" / "rejected_connections")
+  implicit val ns = io.metrics
```

Comment:
I might be mistaken, but isn't this changing the address of these metrics?  Before, they were using the io.namespace, now they are using the MetricSystem namespace, wouldn't that mean they lose any iosystem prefix that was there previously?